src/jit/lsra.h

```diff
@@ -380,8 +380,10 @@ struct LsraBlockInfo
     // 0 for fgFirstBB.
     unsigned int         predBBNum;
     BasicBlock::weight_t weight;
-    bool                 hasCriticalInEdge;
-    bool                 hasCriticalOutEdge;
+    bool                 hasCriticalInEdge : 8;
```

Comment:
You're right - I hadn't realized that. That also helps explain why this had no impact on throughput, since there was already padding after these `bool` fields. I'll remove the `: 8`s